tasks/prime.py

```diff
@@ -250,13 +280,49 @@ def update_mto_agent(self):
             **self.user.cert_kwargs,
         )
 
-        resp, success = check_response(resp, "Update MTO Agent", payload)
+        resp, success = check_response(resp, "updateMTOAgent", payload)
 
         if success:
             new_shipment = deepcopy(mto_shipment)
             new_shipment["agents"][0] = resp
             self.replace_prime_data(PrimeObjects.MTO_SHIPMENT, mto_shipment, new_shipment)
 
+    @tag(PrimeObjects.MTO_SERVICE_ITEM.value, "updateMTOServiceItem")
+    @task
+    def update_mto_service_item(self):
+        mto_service_item = self.get_random_data(PrimeObjects.MTO_SERVICE_ITEM)
+        if not mto_service_item:
+            return  # can't run this task
+
+        try:
+            re_service_code = mto_service_item["reServiceCode"]
+        except KeyError:
+            logger.error("⛔️ update_mto_service_item recvd mtoServiceItem without reServiceCode \n{mto_service_item}")
+            return
+
+        if re_service_code != "DDDSIT" and re_service_code != "DOPSIT":
```

Comment:
Another pythonic way to write this would be:

```python
if re_service_code not in ["DDDSIT", "DOPSIT"]:
```

By no means required, it's just a little more concise

Reply:
ooh nice! And matches the error message.